.eslintrc.json

```diff
@@ -204,8 +203,7 @@
             ],
             "format": [
               "camelCase"
-            ],
-            "leadingUnderscore": "require"
+            ]
           },
```

Comment:
Keep the removed rule. Linting is for improving code quality by enforcing strict rules, so better fix issues instead of removing rules.

Reply:
We can talk about specific rules you consider unnecessary.